types/mapcmd/put-all.go

```diff
@@ -60,8 +60,8 @@ func NewPutAll(config *hazelcast.Config) *cobra.Command {
 		if len(mapKeys) != 0 ||
 			len(mapValues) != 0 ||
 			len(mapValueFiles) != 0 ||
-			mapKeyType != "" ||
-			mapValueType != "" {
+			mapKeyType != "string" ||
+			mapValueType != "string" {
 			return hzcerrors.NewLoggableError(nil, fmt.Sprintf("%s is already set, there cannot be additional flags", JSONEntryFlag))
```

Comment:
Why is this change required? Does it make sense to move this validation to its own function?

Reply:
Tried to refactor for clarity. But I did not understand how to move this to a function already in a function. Please let me know if I can make it better.

Reply:
OK, I didn't realize it was already in a function.